src/build/arg/arg_value.rs

```diff
@@ -0,0 +1,77 @@
+use std::fmt::Display;
+
+/// TODO
+#[derive(Debug, Default, Clone)]
+pub struct ArgValue<'help> {
+    pub(crate) name: &'help str,
+    pub(crate) about: Option<&'help str>,
+    pub(crate) hidden: bool,
+}
+
+impl Display for ArgValue<'_> {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        if self.name.contains(char::is_whitespace) {
+            write!(f, "{:?}", self.name)
+        } else {
+            write!(f, "{}", self.name)
+        }
```

Comment:
While this gives us a good first-pass, I suspect there are a lot of corner cases with this

Reply:
Yeah, I began thinking, maybe having that doesn't really make sense, I moved it from:
https://github.com/clap-rs/clap/blob/0105e6f0b8d9923a0d84d9aba7425489d30869d5/src/output/help.rs#L600-L604
But maybe we should just move it back and not implement Display for ArgValue, don't really see an actual usecase for this, because you need different escaping everywhere (e.g. different shells for completion).